packages/abi-typegen/src/templates/common/_header.hbs

```diff
@@ -7,6 +7,10 @@
 
 /*
   Fuels version: {{FUELS}}
+{{#if FORC}}
   Forc version: {{FORC}}
+{{/if}}
+{{#if FUEL_CORE}}
   Fuel-Core version: {{FUEL_CORE}}
+{{/if}}
```

Comment:
Initially, these were the SDK-supported versions of each tool, but I can see how this PR makes more sense. 👍

After all, we already have the `fuels` version anyway.